hibernate-core/src/main/java/org/hibernate/engine/jdbc/connections/spi/HANAMultiTenantConnectionProvider.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.engine.jdbc.connections.spi;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.hibernate.HibernateException;
+import org.hibernate.MultiTenancyStrategy;
+import org.hibernate.cfg.AvailableSettings;
+import org.hibernate.engine.config.spi.ConfigurationService;
+import org.hibernate.engine.jdbc.connections.internal.ConnectionProviderInitiator;
+import org.hibernate.service.spi.Configurable;
+import org.hibernate.service.spi.ServiceRegistryAwareService;
+import org.hibernate.service.spi.ServiceRegistryImplementor;
+import org.hibernate.service.spi.Stoppable;
+
+public class HANAMultiTenantConnectionProvider extends AbstractMultiTenantConnectionProvider
+		implements ServiceRegistryAwareService, Stoppable, Configurable {
```

Comment:
This class should be removed.  We do not provide specific MultiTenantConnectionProvider implementations

Reply:
Although I could see this generalized into a provider for multi-tenancy Connections based on providing the tenant name as URL parameter or property.

Reply:
OK, I'll remove it.